Docs/Book/C++Examples/example8.cpp

```diff
@@ -0,0 +1,21 @@
+//
+// Modifying molecules example8.cpp
+
+#include <iostream>
+
+#include <GraphMol/GraphMol.h>
+#include <GraphMol/SmilesParse/SmilesParse.h>
+#include <GraphMol/MolOps.h>
+
+int main( int argc , char **argv ) {
+
+  RDKit::ROMOL_SPTR mol1( RDKit::SmilesToMol( "CCO" ) );
```

Comment:
As this is an educational document, would it be useful to add a comment pointing this out and explaining it? I would have used RDKit::ROMol* (this is used for example in the tests). If ROMOL_SPTR is the best practice, it would be good to encourage it.